app/test_main.py

```diff
@@ -1,3 +1,47 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
+            (0, 0, [0, 0]),
+            (3, 12, [0, 0]),
+            (12, 4, [0, 0]),
+            (14, 14, [0, 0]),
+            (-20, -15, [0, 0]),
```

Comment:
The expected results for these test cases do not seem to logically follow from the input values. Please ensure that the expected results are correct according to the logic of the `get_human_age` function.